lib/components/narrative/narrative-itineraries.js

```diff
@@ -193,7 +227,7 @@ class NarrativeItineraries extends Component {
                 routingType='ITINERARY'
                 showRealtimeAnnotation={showRealtimeAnnotation}
                 sort={sort}
-                {...this.props}
+                {...itineraryBodyProps}
```

Comment:
How many of the props are really needed here? In general, I think spreading props to child components should try to be avoided if possible.

Reply:
Addressed, hopefully, in https://github.com/opentripplanner/otp-react-redux/pull/339/commits/3b2b8801f7c708392ca787f0401dc08f6be9103b.